src/Widget/Block.php

```diff
@@ -49,13 +51,22 @@ class Block extends Widget
      */
     private $renderInPlace = false;
 
+    /**
+     * @param WebView $webView
+     */
+    private $webView;
+
+    public function __construct(WebView $webView)
+    {
+        parent::__construct($webView);
+        $this->webView = $webView;
+    }
+
     /**
      * Starts recording a block.
      */
     public function init(): void
     {
-        parent::init();
-
         ob_start();
         ob_implicit_flush(0);
     }
```

Comment:
Can `init()` be removed and code moved to __construct() ?

Reply:
@machour Definitely not - instantiating widget is not the same as rendering it.